source/diagrams/manage-diagrams/create-diagrams.txt

```diff
@@ -10,4 +10,46 @@ Create Diagrams
    :depth: 1
    :class: singlecol
 
-Create diagrams.
\ No newline at end of file
+You can create new diagrams on the Mapping tab of Relational Migrator.
+You can also create multiple diagrams and navigate between them to view
+specific entities of your data model. By default, Relational Migrator
+creates the ``Main Diagram`` diagram with a Relational view and a
+MongoDB view.
+
+About this Task
+---------------
+
+Creating diagrams allows you to visualize tables, foreign key
+relationships, and data types in different formats. After you create a
+diagram, you can :ref:`clone <rm-clone-diagrams>`, :ref:`rename
+<rm-rename-diagrams>`, and :ref:`delete <rm-delete-diagrams>` your new
+diagram. 
+
+You must have at least one digram on your ribbon tab. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Click the plus sign button on the right of the workspace tab 
```

Comment:
**Suggestion**

I like to use the icon role when referring to UI elements such as these, it can help with non-english readers. We also have the ability to match the exact leafy green icons that this app was built with, I put [an edit into the style guide which lists out all available icons ](https://www.mongodb.com/docs/meta/reference/icon/#rendered-icon-table):). 

```suggestion
   .. step:: Click the :icon-lg:`Plus` button on the right of the workspace tab 
```

Reply:
Done!